docs/user_guide/annotation_tool.md

```diff
@@ -1,153 +1,67 @@
+   - Once a standardized variable is selected the tool will automatically infer the data type
 
-# Build for production and launch server
-$ npm run build
-$ npm run start
+!!! tip "When to manually select data type"
+    You need to manually select the data type (Categorical or Continuous) in two cases: when your column doesn't match any standardized variable, and when you map to a multi-column measure standardized variable like "Assessment tool".
+   
+![Annotation tool multi-column measures step screenshot](../imgs/annotate/multi_column_measures.png)
 
-# Generate static project
-$ npm run generate
-```
-### Deployment
+- **Multi-column Measures**: If you have multiple columns for the same measure like assessment tool, group them together
 
-To deploy the static build on GH pages, run
 
-```bash
-npm run generate
-npm run deploy
-```
+### 3. Value Annotation
+![Annotation tool value annotation step screenshot](../imgs/annotate/value_annotation.png)
 
-See the [Nuxt documentation](https://nuxtjs.org/deployments/github-pages/) for more details.
+Use the sidebar to navigate between your columns and annotate their values:
 
-### Developer information
+!!! note "Understanding sidebar sections"
+    The sidebar groups your columns by annotation progress: **Annotated** contains columns mapped to standardized variables, **Unannotated** contains columns only with assigned data types, and **Other** contains columns that still need data type assignment.
 
-**use node v16.x LTS**!
+**For continuous columns** (Age, scores):
+- Select format (float, int, etc.)
+- Add units description ("years", "points")
 
-:warning: 
-The Annotation Tool is built with the Nuxt framework and currently depends on Nuxt2. 
-Nuxt2 does not support node versions beyond the v16 LTS (see e.g. [this Github issue](https://github.com/nuxt/nuxt/issues/10844)).
-If you want to run the tool locally, make sure you are using node v16.x.
-A good way to manage different node versions is to use the [node version manager](https://github.com/nvm-sh/nvm) tool.
+!!! info "Units vs. Format"
+    **Format** refers to how the numbers in your data are structured (e.g., "float" for decimal numbers like 25.5, "int" for whole numbers like 25) whereas **Units** describe what the numbers represent (e.g., "years" for age, "points" for test scores, "mg/dL" for measurements).
 
-## Annotation Workflow
+**For categorical columns** (Sex, Diagnosis):
+- Map each value to standardized terms
+- Add descriptions for each unique value
 
-1. Upload data table (and/or data dictionary)
-2. Linking columns you want annotated with Neurobagel metadata categories
-3. Annotate the values of the those categirzed columns
-4. Download an annotated version of a BIDS-style data dictionary for your dataset
+**Handle missing or unmappable values**: Use "Mark as missing" for values that are absent, not applicable, or don't match any standardized term options
```

Comment:
General question about the value annotation instructions: do we need to specify in the value annotation section which metadata will be 'annotatable' only for columns that have been mapped to a standardized variable? (format, missing values, etc.)